lib/apexcharts/options/legend.rb

```diff
@@ -1,24 +1,25 @@
 module ApexCharts
   class LegendOptions < ::SmartKv
     optional *%i[
-      containerMargin
```

Comment:
The site might not be quite updated. `containerMargin` is a valid option: https://github.com/apexcharts/apexcharts.js/blob/512148c1462a4c8c9d3beca4988a6cc36af4b99b/types/apexcharts.d.ts#L580